clemens-huebner/amos-ss16-proj8
# 60 add organizer

It is about adding registered user to organizer Collection to be organizer.
to test it : log in with Admin
go to Add Organizer in the menu 
put user email 
to check it go to baasbox / Collection organizer
@trungluongquang I integrated  function to check if the putting email is already registered or not, it's also part of my user story and it make no sense to remove it.
